product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+
+PEMx renders insightful charts that empower users to actively monitor each metric's behavior. PEMx provides a streamlined path for users to take prompt and informed actions for the generated alerts. This cohesive monitoring and alerting system ensures comprehensive oversight and prompt response within the BigAnimal environment.
+
+Upon logging in, PEMx monitoring widgets are displayed on the Overview page on the project summary page. These widgets are available only for the deployed PostgreSQL clusters and are not available for the clusters in the provisioning process. They provide a high level key information on:
+- Memory
+- CPU
+- Storage usage percentages
+- Transactions per second
+- Database size
+
+To see more in-depth metrics specific to the PostgreSQL cluster click on any widget on the overview page. It redirects to the **Monitoring and Logging** tab of the cluster page.
+
+![Monitoring and logging tab](images/monitoring_and_logging_tab.png)
+
+To view the **Monitoring and Logging** tab from the BigAnimal portal:
+
+1. Go to **Clusters** in the left navigation of BigAnimal portal. 
+2. Select any ready cluster. 
+3. Select **Monitoring and Logging** tab on the cluster detail page.
+
+The **Monitoring and Logging** tab displays the detailed PEMx monitoring metrics specific to that cluster in the form of charts:
+
+- [Single value statistics](#single-value-statistics)
+- [Historical charts](#historical-charts)
+
+## Single value charts
+
+These charts displays a specific single value based on the last value in the selected time interval. For example, if you choose `last N`, it displays the current value and for the specific time interval, it displays the value at the end of the selected time interval. These charts display key metrics in text, gauges, or pie and donut form. It provides a concise snapshot of information such as: 
+  - **Memory** (Gauge chart) &mdash; Displays the percentage of memory utilized by the node hosting the PostgreSQL cluster.
+  - **CPU** (Gauge chart) &mdash; Displays the percentage of CPU utilized for the PostgreSQL cluster.
+  - **Storage** (Gauge chart) &mdash; Displays the percentage of the storage volume utilized by the PostgreSQL cluster.
+  - **Storage** (Donut chart) &mdash; Displays the units of the storage volume utilized and available for the PostgreSQL cluster.
+  - **Active connections** (Text value) &mdash; Displays the current number of connections between the client applications and the PostgreSQL cluster.
+  - **Disk throughput** (Text value) &mdash; Displays the amount of data transferred to and from the disk per second.
+  - **Transactions per second** (Text value) &mdash; Displays the total number of transactions executing within a second for the PostgreSQL cluster.
+  - **Database size** (Text value) &mdash; Displays the amount of storage volume utilized by the database.
```

Comment:
WDYT if we generalize as potentially there can be multiple databases in the same Postgres instance?